Makefile

```diff
@@ -5,7 +5,7 @@ ARO_IMAGE_BASE = ${RP_IMAGE_ACR}.azurecr.io/aro
 E2E_FLAGS ?= -test.timeout 180m -test.v -ginkgo.v -ginkgo.noColor
 
 # fluentbit version must also be updated in RP code, see pkg/util/version/const.go
-FLUENTBIT_VERSION = 1.9.1-1
+FLUENTBIT_VERSION = 1.9.4-1
```

Comment:
No corresponding RP change?

Reply:
as I understand, `const.go` influences what version clusters are running vs what the RP is running (determined by the RP-Config repo). I modified the config for int, then followed the new deployment procedure to build all artifacts and deploy that updated configuration. If we also want to bump the fluentbit version run by clusters, I can include that here as well.

Reply:
In the past we had a regression with the `fluentbit` version image we put on customer clusters, resulting in logging breaking for all new clusters and existing clusters we PUCMed.  We currently don't have e2e tests on ensuring logging is working as expected and not breaking fluentbit.  

Our security vulnerabilities are flagged on the RP VMSS, not fluentbit so as a side-step we've only been updating the RP image due to higher priorities (Hive, OCM, etc.).  

If we want to bump the fluentbit version in `pkg/util/version/const.go` it will require a bit of soak testing with an updated image and ensuring there are no gaps in logs.  I'm fine with updating it, but we've just been cautious not to break logging in the past and only updating RP-Config.